src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,86 @@
+import { CSSProperties, KeyboardEvent } from "react";
+import { Close } from "@material-ui/icons";
+import styled from "styled-components";
+import _IconButton from "../IconButton";
+
+interface DialogProps {
+  open: boolean;
+  disableCloseButton?: boolean;
+  onClose?: () => void;
+  children?: React.ReactNode;
+  className?: string;
+  style?: CSSProperties;
+}
+
+export default function Dialog({
+  open,
```

Comment:
[nitpick] - do you wanna set the prefix `is-` for boolean properties?

Reply:
yes, I don't mind to add it, I followed material ui library.